src/imgtools/dicom/__init__.py

```
@@ -7,9 +10,15 @@
     load_seg_dcm,
     rtstruct_reference_uids,
 )
-from .utils import find_dicoms, lookup_tag, similar_tags, tag_exists
+from .interlacer import GroupBy, Interlacer
+from .utils import lookup_tag, similar_tags, tag_exists
+
+from .crawl import Crawler, CrawlerSettings  # fmt isort:skip
```

Ruff (0.8.2)

Redefinition of unused `Crawler` from line 1

Remove definition: `Crawler`

(F811)